ipld-future/README.md

```diff
@@ -0,0 +1,179 @@
+# IPLD Spec v1
+
+Editor: Nicola Greco, MIT
+
+> This specification defines a data model and a naming scheme for linking data with cryptographic hashes.
+>
+> InterPlanetary Linked Data (IPLD) is an information space of inter-linked data, where content addresses and links are expressed using the content's cryptographic hash. IPLD is designed to universally address and link to any data so that the data does not require a central naming authority, the integrity of the data can be verified, and untrusted parties can help distribute the data. This specification describes a data model for structured data and a naming scheme to point to data and subsets of the data. These design goals make it different from earlier data models such as JSON or RDF, and naming schemes such as NI [[RFC6920]](https://tools.ietf.org/html/rfc6920), or Magnet URI.
```

Comment:
@RichardLitt that is true, I need to check with Juan on how we want to clearly define IPLD. Technically, information space is a valid wording (the web, for example, is an information space; an information space is made of linked documents) - maybe we can just replace that with

> IPLD is a web of data, where data is .. and so on

What do you think?